app/src/main/java/com/x8bit/bitwarden/data/auth/repository/util/UserStateJsonExtensions.kt

```diff
@@ -174,7 +174,7 @@ fun UserStateJson.toUserState(
                     isBiometricsEnabled = isBiometricsEnabledProvider(userId),
                     vaultUnlockType = vaultUnlockTypeProvider(userId),
                     needsMasterPassword = needsMasterPassword,
-                    hasMasterPassword = decryptionOptions?.hasMasterPassword != false,
+                    hasMasterPassword = decryptionOptions?.hasMasterPassword ?: false,
```

Comment:
@david-livefront want to make sure this is correct previously what was happening was:

```
decryptionOptions.hasMasterPassword == null // true
null != false // true so these users were being marked as having a master password. 
```

Reply:
want to make sure this does not break some other scenario where we purposely want the default to be true

Reply:
I believe this will break other scenarios, if the `decryptionOptions` are null then that means the user should have a master password.